python/ray/data/aggregate.py

```diff
@@ -90,13 +203,17 @@ def __init__(self, on: Optional[KeyFn] = None):
 class Min(_AggregateOnKeyBase):
     """Defines min aggregation."""
 
-    def __init__(self, on: Optional[KeyFn] = None):
+    def __init__(self, on: KeyFn = None, ignore_nulls: bool = True):
```

Comment:
Shouldn't the typing be Optional[KeyFn]?

Reply:
That's my preference, but I saw that @ericl removed all of the `Optional[]` typing on this `KeyFn` in an earlier PR, so I'm trying to standardize on a choice rather than push my preference. But we should note that if we were to run a Python type-checker on all of this, not having the `Optional[]` type would cause this to fail.